nav.php

```diff
@@ -36,6 +32,9 @@
 							</ul>
 						</li>";
                     }?>
+			<li class="hide"><a style="margin-right: 10px;">|</a></li>
```

Comment:
#changed the nav layout by moving the username next to the logout to adhere to external consistency